### ISSUE TITLE
Make the layout in grid display editable, the groups resizable.

### ISSUE THREAD

**Issue by mzso** [Quicksaver/Tab-Groups#398]
Hi!

I grew to like the grid view because it always uses the whole screen area, and the layout is never ruined by resized windows.
I think it would be even more efficient if I could freely resize tab groups/sections so those groups that are more important would have more space available.

**Comment by Quicksaver** [comment]
I don't believe I will add this any time soon at least, to be completely honest, I'm only still considering it because a large number of users seem to want this.

I do see how an enhanced interface like that could be useful, the problem I see is with creating it. It would need a whole new positioning system, and frankly I look at the issue and think it's a whole lot of work for not that big of a change.

Again, that's not to say I don't appreciate that it could be useful as you mentioned, and it may happen in the future, but not any time soon as I'd rather give priority to many other issues at the moment. I hope you understand.

**Comment by mzso** [comment]
Meanwhile something much simpler migh be the configurability of  how the groups are sized? Right now I have to big ones, two smaller ones, and three tiny ones.

**Comment by Quicksaver** [comment]
> Meanwhile something much simpler migh be the configurability of how the groups are sized? Right now I have to big ones, two smaller ones, and three tiny ones.

That's all completely automated according to your screen size. You can make them all the same size (except for the bare minimum differences required so they all fit on screen) by disabling the appropriate preference in the add-on's options. I think the ability to configure these sizes goes hand in hand with your original request.
